### PR TITLE
Implement scale word realization

### PR DESCRIPTION
**Pull request by frostburn**
ref #188

### CODE REVIEW

**Comment by frostburn** [comment]
The name is a bit long. Would `realizeWord` be clear enough?

**Comment by inthar-raven** [comment]
> The name is a bit long. Would `realizeWord` be clear enough?

I think so.

**Review by inthar-raven — APPROVED** [review]
I just renamed the function, and otherwise it looks good to me. Could you review my changes just in case?

File: BUILTIN.md [hunk 1]
```diff
@@ -563,6 +566,9 @@ Obtain a copy of the current/given scale with random variance added.
 ### rank2(*generator*, *up*, *down = 0*, *period = 2*, *numPeriods = 1*)
 Generate a finite segment of a Rank-2 scale generated by stacking the given generator against the given period (or the octave `2/1` by default). `up` and `down` must be multiples of `numPeriods`.
 
+### realizeScaleWord(*word*, *sizes*, *equave = niente*)
```
Comment (frostburn) [inline BUILTIN.md:569]: Remember to `npm run autodoc`.

**Pull request merged**